bouncer.go

```diff
@@ -221,16 +255,24 @@ func (bouncer *Bouncer) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 	// Right here if we cannot join the stream we forbid the request to go on.
 	if bouncer.crowdsecMode == configuration.StreamMode || bouncer.crowdsecMode == configuration.AloneMode {
 		if isCrowdsecStreamHealthy {
+			logger.Debug(fmt.Sprintf("ServeHTTP:stream is healthy"))
 			bouncer.next.ServeHTTP(rw, req)
 		} else {
 			logger.Debug(fmt.Sprintf("ServeHTTP isCrowdsecStreamHealthy:false ip:%s", remoteIP))
-			rw.WriteHeader(http.StatusForbidden)
+			if bouncer.forbidOnFailure {
```

Comment:
this is your function `handleBanResponseSoft`

Reply:
done, coming soon